index.html

```diff
@@ -22,7 +22,13 @@ <h1>Welcome to the student showcase!</h1>
         <div>TWO</div>
         <div>THREE</div>
         <div>FOUR</div>
-        <div>FIVE</div>
+        <div>
+            <h3>Damian Clay-Downing</h3>
+            <p>"I can accept failure, everyone fails at something. But I can't accept not trying. Micheal Jordan"</p>
+            <img src="https://i.gifer.com/PD4.gif" alt="">
+            <a href="https://www.linkedin.com/in/damian-clay-downing-8ba89155/"><li>LinkedIn</a>
```

Comment:
`<li>` stands for `List Item`, which means that `<li>` should _only_ be found in a list of some sort (either ordered or unordered). Also be sure to close those tags that you do start.